migrations/20200824143059-new-order-statuses.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.query(
+      `UPDATE "Orders"
+  SET "status" = 'PLEDGED'
+  FROM "Collectives"
+  WHERE "Orders"."status" = 'PENDING'
+  AND "Orders"."PaymentMethodId" IS NOT NULL
```

Comment:
Shouldn't that be `IS NULL`? I don't think pledges have payment methods attached

```suggestion
  AND "Orders"."PaymentMethodId" IS NULL
```